scripts/jenkins/build_clang_gridtools.sh

```diff
@@ -86,3 +93,16 @@ cmake --build . --parallel ${PARALLEL_BUILD_JOBS}
 
 # Run unittests
 ctest -VV -C ${build_type} --output-on-failure --force-new-ctest-process
+
+if [ -z ${RUN_PERFTETS+x} ]; then
+  echo "do not run performance tests"
+else
+  target=cuda
```

Comment:
this will change to the above once we merge to have control in jenkins. This cannot be done in the PR setup